eclipse-che/che-server
# Provide scm info in factory v2

### What does this PR do?
Provide SCM project details required for dashboard from Che-server in a separate field on the Factory.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19816


### How to test this PR?
1) deploy test image `mshaposh/che-server:nightly`
2) accept factory with devfile v2.
3) make sure factory contains SCM details field


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
